README.md

```diff
@@ -108,7 +109,7 @@ Lists of issues addressed per release can be found in [JIRA](https://jira.spring
 Here are some ways for you to get involved in the community:
 
 * Get involved with the Spring community on the Spring Community Forums.  Please help out on the [forum](https://forum.spring.io/) by responding to questions and joining the debate.
-* Create [JIRA](https://jira.spring.io/browse/AMQP) tickets for bugs and new features and comment and vote on the ones that you are interested in.  
+* Create [JIRA](https://jira.spring.io/browse/AMQP) tickets for bugs and new features and comment and vote on the ones that you are interested in.
```

Comment:
Not related to this change, but we should look into changing this paragraph anyway: no `forum` any more, no JIRA.
But feel free to fix it right now!